examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -208,3 +213,73 @@
         fontsize=20
     )
 plt.show()
```

Comment:
This graph is too small. Difficult to see.

Reply:
Done.